src/crate/crash/command.py

```diff
@@ -235,9 +235,9 @@ def exit(self):
     @noargs_command
     def _show_tables(self, *args):
         """ print the existing tables within the 'doc' schema """
-        self._exec("""select format('%s.%s', schema_name, table_name) as name
+        self._exec("""select format('%s.%s', table_schema, table_name) as name
```

Comment:
updated the check. however i had to define the TABLE_SCHEMA_MIN_VERSION twice, and also have the feeling that i needed to duplicate too much SQL code :)
let me know if you have suggestions on how to better structure it